### PR TITLE
Telescope enabled_methods

### PR DESCRIPTION
**Pull request by peterfication**
Fixes https://github.com/johmsalas/text-case.nvim/issues/92

### CODE REVIEW

File: tests/environments/lsp.lua [hunk 1]
```diff
@@ -8,6 +8,7 @@ end
 function M.init()
   vim.cmd([[set runtimepath=$VIMRUNTIME]])
   vim.opt.runtimepath:append(M.root())
+  vim.opt.swapfile = false
```
Comment (johmsalas) [inline tests/environments/lsp.lua:11]: nice

File: tests/textcase/telescope/options/enabled_methods_spec.lua [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      enabled_methods = { "to_snake_case" },
+    })
+    require("telescope").load_extension("textcase")
+
+    local buf = vim.api.nvim_create_buf(false, true)
+    vim.api.nvim_command("buffer " .. buf)
+    vim.api.nvim_buf_set_lines(0, 0, -1, true, { "LoremIpsum" })
+  end)
+
+  it("should not register other keymappings", function()
```
Comment (johmsalas) [inline tests/textcase/telescope/options/enabled_methods_spec.lua:21]: Did you mean `keymapping` in the context of Telescope enabled methods?

Comment (peterfication) [reply tests/textcase/telescope/options/enabled_methods_spec.lua:21]: Ah copy&paste mistake from the other enabled_methods spec 🙈

Comment (peterfication) [reply tests/textcase/telescope/options/enabled_methods_spec.lua]: I updated it.

**Review by johmsalas — APPROVED** [review]

**Pull request merged**